### PR TITLE
Feature/create stream 4 cluster

### PR DESCRIPTION
**Pull request by gangtao**
Adding shards and replica configuration for stream creation.
Updating test cases to support cluster proton

### CODE REVIEW

File: python/timeplus/dbapi.py [hunk 1]
```diff
@@ -151,20 +151,27 @@ def __exit__(self, *exc):
...
+        except:
+            return ExternalStream(env=self.env).name(name).get()
 
     def _get_view(self, name):
         return View(env=self.env).name(name).get()
 
     def _list_table(self):
-        return Stream(env=self.env).list()
+        streams =  Stream(env=self.env).list()
+        estreams = ExternalStream(env=self.env).list()
```
Comment (ye11ow) [inline python/timeplus/dbapi.py:172]: do we want to list external table?

**Review by ye11ow — APPROVED** [review]

File: python/timeplus/dbapi.py (hunk 1 above)
Comment (gangtao) [reply python/timeplus/dbapi.py:172]: Yes, that is good to think about, so far the python SDK did not include external table yet, I probably will add it in next PR

**Review by jovezhong — APPROVED** [review]

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**